Src/BootCamp.Chapter/HwFunctions.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+
+namespace BootCamp.Chapter
+{
+    public class HwFunctions
```

Comment:
The name of this class was requested to be Lesson3.

Reply:
Made the changes.